### ISSUE TITLE
Simurgh hits too soft

### ISSUE THREAD

**Issue by Steam-arch** [EdenServer/community#5090]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducible steps. (No "as the title says" posts please. Provide reproducible steps even if it seems like a waste of time.)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

taru drg/whm eden damage taken:
![image](https://user-images.githubusercontent.com/69089249/170874346-5085b424-4b69-4348-a2ee-f33f3687e6d1.png)
hits around 35-86 on eden

mithra drg/whm same gear + food on retail:
![image](https://user-images.githubusercontent.com/69089249/170874387-35f4b118-638d-420d-a019-e6c661a810bf.png)
![image](https://user-images.githubusercontent.com/69089249/170874401-e18e2f28-4c3e-4e11-aa33-ed915b746fc0.png)
![image](https://user-images.githubusercontent.com/69089249/170874439-7d9d0fe7-e5d8-4788-a361-e7d89051b4de.png)
![image](https://user-images.githubusercontent.com/69089249/170874451-f9612ace-51dd-4801-8519-eb8d48809233.png)
![image](https://user-images.githubusercontent.com/69089249/170874467-42c41e3c-5d70-486c-869d-a8f138cf2fca.png)
![image](https://user-images.githubusercontent.com/69089249/170874482-82dfe974-1413-4256-8e1c-61ab3e32a35e.png)
![image](https://user-images.githubusercontent.com/69089249/170874503-e83e3a95-0923-41b9-bbdf-6ca8c5280718.png)
![image](https://user-images.githubusercontent.com/69089249/170874572-b5238e34-f3b7-4c0b-93f9-4a8cb01d89a8.png)
(tried to take some of these from the start so no minuet)
hits around 80-174 on retail

**Comment by ThoseWhoFight** [comment]
Nice, interesting. I've always thought Simurgh was way too easy here compared to how I remember and also other servers. I feel like he's not evasive enough too still possibly. Or something defensively is off, but I've seen the video you are linking SS from so maybe I'll also take a look later.

**Comment by Steam-arch** [comment]
@ThoseWhoFight 
you are right about the eva. ty for pointing it out. ill check it out later

**Comment by Steam-arch** [comment]
Adding the Tamika-video source: 
https://www.youtube.com/watch?v=J2EDrS6ntj4
https://www.youtube.com/watch?v=pN7y6uRL8ic&t=96s

**Comment by 59blargedy** [comment]
Resolved in June 13, 2022 update.
